### PR TITLE
Add clipper.exchange Coves

### PR DESCRIPTION
**Pull request by amalashkevich**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — CHANGES_REQUESTED** [review]
Thank you @amalashkevich 
Here are some minor feedbacks about tests

File: models/clipper/arbitrum/clipper_arbitrum_schema.yml [hunk 1]
```diff
@@ -131,3 +130,38 @@ models:
...
+      blockchain: arbitrum
+      sector: dex
+      project: clipper
+      contributors: 0xRob, amalashkevich
+    config:
+      tags: ['arbitrum','clipper','trades','dex','0xRob', 'amalashkevich']
+    description: >
+        Clipper trades on Arbitrum
+    columns:
+      - *blockchain
```
[the remark this answers is not in the document]
Comment (amalashkevich) [reply models/clipper/arbitrum/clipper_arbitrum_schema.yml]: Thanks, fixed

File: models/clipper/polygon/clipper_polygon_schema.yml [hunk 2]
```diff
@@ -145,6 +145,55 @@ models:
...
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
+      - check_dex_seed:
+          blockchain: polygon
+          project: clipper
+          version: coves_v1
```
[the remark this answers is not in the document]
Comment (amalashkevich) [reply models/clipper/polygon/clipper_polygon_schema.yml]: Thanks, fixed

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @amalashkevich

**Review by jeff-dude — APPROVED** [review]
thanks all

**Pull request merged**